MAUI/LinearGauge/pointers.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: Interaction in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about the interactions in Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Pointers in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The pointer is used to indicate values on a scale. The Linear Gauge control has three types of pointers:
+
+[`Bar pointer`]()
+[`Shape Pointer`]()
+[`Content pointer`]()
+
+All the pointers can be customized as needed. You can add multiple pointers to the gauge to point multiple values on the same scale. The value of the pointer is set using the Value property.
```

Comment:
The value of the pointer is set using the `Value` property.

Please highlight the property names